Example/BonMot/BONSpecialGenerator.swift

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env swift
+
+//
+//  BONSpecialGenerator.swift
+//  BonMot
+//
+//  Created by Zev Eisenberg on 7/16/15.
+//
+
+import Foundation
+import AppKit
+
+// Please keep this array sorted
+let specialCharacters: [unichar] = [
+    0x0009,
+    0x000A,
+    0x0020,
+    0x00A0,
+    0x2002,
+    0x2003,
+    0x2007,
+    0x2009,
+    0x200A,
+    0x200B,
+    0x2011,
+    0x2012,
+    0x2013,
+    0x2014,
+    0x2026, // HORIZONTAL ELLIPSIS
```

Comment:
Commenting helps a little here, but as you say, one could just look in .m file.  I have no strong feelings either way.

If I were to put on my "let's over-engineer-this" hat, I could suggest that the comment could be some form that would allow the script to sanity-check the name it derives against the name that the author intended.  But it's not a hat day.